openstack/db/v1/configurations/testing/requests_test.go

```diff
@@ -235,3 +235,21 @@ func TestGetGlobalParam(t *testing.T) {
 
 	th.AssertDeepEquals(t, expected, param)
 }
+
+func TestAttachConfigGroup(t *testing.T) {
+	th.SetupHTTP()
+	defer th.TeardownHTTP()
+	fixture.SetupHandler(t, resURL, "PUT", "", "", 202)
+
+	_, err := configurations.AttachConfigGroup(fake.ServiceClient(), configID, "zzz").Extract()
```

Comment:
It looks like the function parameters have gotten mixed up here.

In the function signature you have:

```go
AttachConfigGroup(client *gophercloud.ServiceClient, id string, configID string)
```

So I would think that `configID` and `zzz` would be swapped?

Reply:
yep, looks like that's wrong .. should it invoke also instance creation and send real id there?